### ISSUE TITLE
Batch Support for Ubuntu 16.04 Ending

### ISSUE THREAD

**Issue by alfpark** [Azure/Batch#106]
Azure Batch will retire support for Canonical UbuntuServer 16.04-LTS to follow the publisher ending standard support in April 2021. If your workload utilizes Batch pools based on this OS, either via Marketplace images (directly from Canonical or derived from by other publishers) or your own custom images, then you will need to take action. Please migrate your workload to a publisher supported version of Canonical UbuntuServer such as 20.04-LTS (recommended) or 18.04-LTS. On 2021-05-21, Azure Batch will remove support for this compute node agent. Afterwards, create pool and resize up operations may fail. Customers who continue to use these pools past the indicated Batch support End of Life will be subject to potential security risks.

For more information, please see:

- [Azure Batch VM Size and Image Guide](https://docs.microsoft.com/azure/batch/batch-pool-vm-sizes)
- [Azure Batch Best Practices](https://docs.microsoft.com/azure/batch/best-practices)

**Comment by galiana** [comment]
Currently I'm using the following supported image:
Offer: ubuntu-server-container-rdma -  SKU: 16-04-lts
I have listed the supported Images on my batch service to check wich SKU supports RDMA, and Noticed only the SKU 16-04-lts supports RDMA.
Should I expect a new image for Ubuntu 18-20 to be available soon? Otherwise I can't follow your recommendations, isn't it?

**Comment by alfpark** [comment]
You are correct. We are in the process of publishing a new image for this particular offer which will be based on 20.04-LTS.

**Comment by galiana** [comment]
Great. I'd really appreciate if you could update this "Issue" once the image is published.

**Comment by vincentlabonte** [comment]
The image should now be published and available for new pools in all regions.

**Comment by mphoboon** [comment]
Hi we are using the linux data science image (linux-data-science-vm-ubuntu) which is running on 16.04. Is there a replacement for the data science image?

**Comment by vincentlabonte** [comment]
@mphoboon, You can take a look at the [Data Science Virtual Machines](https://azure.microsoft.com/en-us/services/virtual-machines/data-science-virtual-machines/). The [Linux version](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/microsoft-dsvm.ubuntu-1804) uses Ubuntu 18.04 which, I think, is the replacement for `linux-data-science-vm-ubuntu` you mentionned.

**Comment by mphoboon** [comment]
@vincentlabonte Thank you. I found the one you mentioned but I cannot find a way to select it when I was trying to create a new pool. We used to select microsoft-dsvm as the publisher but we cannot find it anywhere.

**Comment by mphoboon** [comment]
@vincentlabonte I just learned that in the create new pool UI, I have to check 'Enable unverified image' to see the publisher. Thank you.
